edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/AssemblyInfo.cs

```diff
@@ -4,3 +4,4 @@
 [assembly: InternalsVisibleTo("Microsoft.Azure.Devices.Edge.Hub.Core.Test")]
 [assembly: InternalsVisibleTo("Microsoft.Azure.Devices.Edge.Hub.Mqtt.Test")]
 [assembly: InternalsVisibleTo("Microsoft.Azure.Devices.Edge.Hub.E2E.Test")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
What's this for?

Reply:
To allow mocking Internal interfaces with Moq